### PR TITLE
Pods 8398

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/transformations/UserAnswersToETMP/API1830.scala [hunk 1]
```diff
@@ -63,110 +63,122 @@ object API1830 extends Transformer {
...
-      (__ \ Symbol("eventType")).json.put(JsString(s"Event$Event3")) and
+      (pathToEvent \ Symbol("eventType")).json.put(JsString(s"Event$Event3")) and
         readsIndividualMemberDetails and
         (pathPaymentDetails \ Symbol("reasonBenefitTaken")).json.copyFrom(readsTypeOfBenefitEvent3) and
-        (pathPaymentDetails \ Symbol("amountBenefit")).json.copyFrom((pathPaymentDetails \ Symbol("amountPaid")).json.pick) and
-        (pathPaymentDetails \ Symbol("eventDate")).json.copyFrom((pathPaymentDetails \ Symbol("eventDate")).json.pick) and
-        (pathPaymentDetails \ Symbol("freeText")).json.copyFrom(readsFreeTextEvent3.orElse(Reads.pure(JsString("N/A"))))).reduce
+        (pathPaymentDetails \ Symbol("amountBenefit")).json.copyFrom((__ \ Symbol("paymentDetails") \ Symbol("amountPaid")).json.pick) and
+        (pathPaymentDetails \ Symbol("eventDate")).json.copyFrom((__ \ Symbol("paymentDetails") \ Symbol("eventDate")).json.pick) and
+        (pathPaymentDetails \ Symbol("freeText")).json.copyFrom(readsFreeTextEvent3.orElse(Reads.pure(JsString("N/A")))) and
```
Comment (ngmjohnstone) [inline app/transformations/UserAnswersToETMP/API1830.scala:91]: Should we add in TODO comments here to remove the `.orElse` logic once we have received the (anticipated) new schema which corrects these fields to be optional?

File: app/transformations/UserAnswersToETMP/API1830.scala [hunk 2]
```diff
@@ -63,110 +63,122 @@ object API1830 extends Transformer {
...
-      (__ \ Symbol("eventType")).json.put(JsString(s"Event$Event8A")) and
+   (
+      (pathToEvent \ Symbol("eventType")).json.put(JsString(s"Event$Event8A")) and
         readsIndividualMemberDetails and
         (pathPaymentDetails \ Symbol("reasonBenefitTaken")).json.copyFrom(readsPaymentTypeEvent8A) and
         (pathPaymentDetails \ Symbol("typeOfProtection")).json.copyFrom(readsTypeOfProtectionEvent8A.orElse((Reads.pure(JsString("N/A"))))) and
         (pathPaymentDetails \ Symbol("amountLumpSum")).json.copyFrom((pathLumpSumAmountAndDateDetails \ Symbol("lumpSumAmount")).json.pick) and
         (pathPaymentDetails \ Symbol("eventDate")).json.copyFrom((pathLumpSumAmountAndDateDetails \ Symbol("lumpSumDate")).json.pick) and
-        (pathPaymentDetails \ Symbol("freeText")).json.copyFrom(readsTypeOfProtectionReferenceEvent8A.orElse(Reads.pure(JsString("N/A"))))).reduce
+        (pathPaymentDetails \ Symbol("freeText")).json.copyFrom(readsTypeOfProtectionReferenceEvent8A.orElse(Reads.pure(JsString("N/A")))) and
```
Comment (ngmjohnstone) [inline app/transformations/UserAnswersToETMP/API1830.scala:162]: Same as previous

**Review by ngmjohnstone — APPROVED** [review]
Looks good to me, with some comments r.e. the `.orElse` / N/A functionality as discussed earlier today

File: app/transformations/UserAnswersToETMP/API1830.scala (hunk 2 above)
Comment (grantrobson) [reply app/transformations/UserAnswersToETMP/API1830.scala:162]: i THINK for all these comments the todo change will be on a separate tickedt anyway so i wouldn't bother

**Pull request merged**